runtime/Cargo.toml

```diff
@@ -122,11 +182,36 @@ features = ['derive']
 optional = true
 version = '1.0.101'
 
+[dependencies.session]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'srml-session'
+rev = '67b73c468e44839c5809245703d067dc75940076'
+features = ["historical"]
+
 [dependencies.sr-primitives]
 default_features = false
 git = 'https://github.com/paritytech/substrate.git'
 rev = '67b73c468e44839c5809245703d067dc75940076'
 
+[dependencies.sr-staking-primitives]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'sr-staking-primitives'
+rev = '67b73c468e44839c5809245703d067dc75940076'
+
+[dependencies.srml-staking-reward-curve]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'srml-staking-reward-curve'
+rev = '67b73c468e44839c5809245703d067dc75940076'
+
+[dependencies.staking]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
```

Comment:
Is the reward curve package a hard dependency for staking module ? Asking because you suggested we deprioritise this

Reply:
Good point – it is not, but there is no harm using the default one. I think we should deprioritize implementing our own custom reward curve – the current thinking is around exponentially falling rewards over time. That is much more work to implement than just using the existing one.